docs/reference/builder.md

```diff
@@ -92,7 +92,7 @@ instructions.
 Whenever possible, Docker will re-use the intermediate images (cache),
 to accelerate the `docker build` process significantly. This is indicated by
 the `Using cache` message in the console output.
-(For more information, see the [Build cache section](../userguide/eng-image/dockerfile_best-practices.md#build-cache)) in the
+(For more information, see the [Build cache section](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices.md#build-cache)) in the
```

Comment:
should be `https://docker.github.io/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache`